### PR TITLE
add new playbook modprobeib

### PR DESCRIPTION
**Pull request by ahmam**

### CODE REVIEW

File: tasks/modprobeib.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    insertafter: '^alias'
+    line: options ib_ipoib ipoib_enhanced=0
+- name: stop and start service
+  ansible.builtin.service:
+    name: openibd
+    state: stopped
+- name: start service
+  ansible.builtin.service:
+    name: openibd
+    state: started
```
Comment (btravouillon) [inline tasks/modprobeib.yml:14]: Can you use a handler instead? With `ansible.builtin.systemd` and `state: restarted`.

File: tasks/modprobeib.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+- name: disable ipoib_enhanced
+  ansible.builtin.lineinfile:
+    path: /etc/modprobe.d/ib_ipoib.conf
+    insertafter: '^alias'
+    line: options ib_ipoib ipoib_enhanced=0
```
Comment (btravouillon) [inline tasks/modprobeib.yml:6]: There should be a tunable to enable or disable `ipoib_enhanced`. Some users of this role would need IPoIB enhanced (or at least would like to stick to the default values).

File: tasks/modprobeib.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- name: check ipoib_enhanced
+  ansible.builtin.command:
+    cmd: /usr/bin/cat /sys/module/ib_ipoib/parameters/ipoib_enhanced
+  register: pipoib_enhanced
+  failed_when: poib_enhanced.stdout != "0"
+- name: check the mode
+  ansible.builtin.command:
+    cmd: /usr/bin/cat /sys/class/net/ib0/mode
+  register: pmode
+  failed_when: pmode.stdout != "connected"
```
Comment (btravouillon) [inline tasks/modprobeib.yml:24]: There are methodes better than the `command` module to check file content. I'll try to share one.

**Review by btravouillon — CHANGES_REQUESTED** [review]
For readibility, can you ensure to add 1 blank line between each task?

File: tasks/modprobeib.yml (hunk 2 above)
Comment (ahmam) [reply tasks/modprobeib.yml:6]: oui je vais ajouter une condition boolen ipoib_enhanced=true dans les variable

Comment (ahmam) [reply tasks/modprobeib.yml:6]: `infiniband_ipoib_enhanced: True` added to main file

File: tasks/modprobeib.yml (hunk 3 above)
Comment (ahmam) [reply tasks/modprobeib.yml]: oui j'ai une autre proposition:
`- debug: msg="the value of  ipoib_enhanced is {{lookup('file', '/sys/module/ib_ipoib/parameters/ipoib_enhanced') }}"` ou dans unvariable au default 
```
- name: Ensure the ipoib_enhanced
  ansible.builtin.lineinfile:
    path: /sys/module/ib_ipoib/parameters/ipoib_enhanced
    regexp: '^0'
    state: present
```

**Comment by btravouillon** [comment]
Closed in favor of #13.

**Pull request closed without merging**